rc/.eslintrc.json

```diff
@@ -247,14 +267,23 @@
         ],
         "new-parens": 2,
         "newline-after-var": 0,
-        "object-curly-spacing": [0, "never"],
+        "object-curly-spacing": [
+            2,
+            "never"
```

Comment:
This was the default in lint-trap. I too prefer "always" here. I was going with the previous standard, but I'm fine to change this since I think it makes things more readable.

What about the array-spacing rule?